README.md

```diff
@@ -16,171 +17,151 @@
 
 The U.S. Government Department of Commerce, Bureau of Industry and Security (BIS), has classified this software as Export Commodity Control Number (ECCN) 5D002.C.1, which includes information security software using or performing cryptographic functions with asymmetric algorithms. The form and manner of this distribution make it eligible for export under the License Exception ENC Technology Software Unrestricted (TSU) exception (see the BIS Export Administration Regulations, Section 740.13) for both object code and source code.
 
 # Submitting apps to the App Store
 
-If you’re using Themis as your means of encryption within your iOS/macOS app that you’re submitting to the App Store, your encryption falls under the “open source” exception (although if your app is not open source/distributed free of charge, we strongly recommend that you seek legal advice). 
-
-Themis is a free cryptographic library that builds on the existing, community-tested cryptographic instruments (OpenSSL, LibreSSL, BoringSSL, depending on the target platform). It is open source and Apache 2-licensed, with its full source code publicly available online on GitHub. 
+If your application uses Themis and you want to submit it to the Apple App Store, there are certain requirements towards declaring use of any cryptography.
 
-This means that you should indicate that you’re using encryption and only submit annual self-classification reports (use [this handy table](https://help.apple.com/app-store-connect/#/devc3f64248f) to self-check). Read more about [Apple regulations on cryptography](https://medium.com/@cossacklabs/apple-export-regulations-on-crypto-6306380682e1) and [check Apple docs](https://developer.apple.com/documentation/security/complying_with_encryption_export_regulations?language=objc).
+Read about [Apple export regulations on cryptography for Themis](https://docs.cossacklabs.com/themis/regulations/apple-crypto-regulations/) to find out what to do.
 
 # Security
 
 Each change in Themis core library is being reviewed and approved by our internal team of cryptographers and security engineers. For every release, we perform internal audits by cryptographers who don't work on Themis.
 
-We use a lot of automated security testing, i.e. static code analysers, fuzzing tools, memory analysers, unit tests (per each platform), integration tests (to find compatibility issues between different Themis-supported languages, OS and x86/x64 architectures). Read more about our security testing practices in [a blog post](https://medium.com/@cossacklabs/automated-security-testing-56ee1253c1fd).
+We use a lot of automated security testing, i.e. static code analysers, fuzzing tools, memory analysers, unit tests (per each platform), integration tests (to find compatibility issues between different Themis-supported languages, OS and x86/x64 architectures). Read more about our security testing practices in [on Themis security docs](https://docs.cossacklabs.com/themis/security/).
```

Comment:
```suggestion
We use a lot of automated security testing, i.e. static code analysers, fuzzing tools, memory analysers, unit tests (per each platform), integration tests (to find compatibility issues between different Themis-supported languages, OS and x86/x64 architectures). Read more about our security testing practices in [Themis security docs](https://docs.cossacklabs.com/themis/security/).
```
Too many prepositions, I think.

Reply:
it was part of doc name "on Themis security", but i'm ok

Reply:
Oh, right... Then I guess it needed proper capitalization so that it's more clear what part is the title.
```
[...] testing practices in [On Themis security](...) docs.
```
Whatever, I think plain text instead of verbatim title is fine too.